src/addons/presenter.js

```diff
@@ -210,10 +220,14 @@ export default class Presenter extends Component {
     console.warn(`No presenter implements intent “${ intent }”.`)
   }
 
-  render () {
+  view (model) {
     return this.props.children ? React.Children.only(this.props.children) : null
```

Comment:
It makes sure there's only one child. Otherwise it throws with a useful error.

https://facebook.github.io/react/docs/top-level-api.html#react.children.only

Kind of makes me wonder if it's necessary...